chart/kubeapps/README.md

```diff
@@ -153,6 +153,104 @@ If you have dedicated a namespace only for Kubeapps you can completely clean the
 kubectl delete namespace kubeapps
 ```
 
+## FAQ
+
+- [How to install Kubeapps for demo purposes?](#how-to-install-kubeapps-for-demo-purposes)
+- [How to install Kubeapps in production scenarios?](#how-to-install-kubeapps-in-production-scenarios)
+- [How to use Kubeapps?](#how-to-use-kubeapps)
+- [How to configure Kubeapps with Ingress](#how-to-configure-kubeapps-with-ingress)
+  * [Serving Kubeapps in a subpath](#serving-kubeapps-in-a-subpath)
+- [Can Kubeapps be installed in more than one cluster?](#can-kubeapps-be-installed-in-more-than-one-cluster)
+- [How to install Kubeapps without Internet connection?](#how-to-install-kubeapps-without-internet-connection)
+- [Does Kubeapps support private repositories?](#does-kubeapps-support-private-repositories)
+- [Does Kubeapps support Operators?](#does-kubeapps-support-operators)
+- [More questions?](#more-questions)
+
+### How to install Kubeapps for demo purposes?
+
+This snippet shows how to install Kubeapps for exclusively **demo purposes** as explained in [getting started](https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md).
```

Comment:
Rather than duplicating (and maintaining duplicate info), why not just refer to the getting started for his answer?

Reply:
Right... I first thought about having a collection of snippets for each question, but, since we already have a bunch of guides, I opted to directly point to them, as you suggest. However, given that the chart doc is the starting point for some users, I think perhaps it is a good idea to have a full example of how to start running Kubeapps.

I'd rather suggest adding this snippet in the TL;DR or even just add a link to the getting started there.
I'm just considering users who just want to give it a try without thoroughly read the docs.

Reply:
Yep, fair point.